jep/7/README.adoc

```diff
@@ -67,22 +69,68 @@ endif::[]
 
 == Abstract
 
-Due to its negative impact on core maintenance, distribution of the unmaintained `ruby-runtime` plugin will be suspended.
-Distribution of plugins depending on it will be suspended until their dependency is removed.
+Due to its negative impact on core maintenance, distribution of the unmaintained
+link:https://github.com/jenkinsci/ruby-runtime-plugin[Ruby Runtime plugin] and
+link:https://github.com/jenkinsci/jenkins.py[Jenkins Python SDK] will be suspended.
+Distribution of plugins depending on them will be suspended until their dependency is removed.
 
 
 == Specification
 
-`ruby-runtime` will be added to the `artifact-ignores` file of the update center generator so it is no longer available for download on update sites.
-// Perhaps: Add security warning?
+Ruby runtime and Python SDK will be added to the `artifact-ignores` file of the update center generator, so it is no longer available for download on update sites.
+A warning will be added to the update center so that all users of the plugins receive a warning in advance.
+Plugins depending on the either the Ruby runtime or the Python SDK will be also marked for adoption so that users can step up and take over maintenance (by switching to the regular Java runtime) without a 2 weeks waiting delay.
 
-Plugins with a mandatory dependency on `ruby-runtime` will be added to the same file, as they will not be installable without `ruby-runtime` being available.
+Plugins with a mandatory dependency on the runtimes will be added to the same file, as they will not be installable without the respective runtimes being available.
+The multiple Git repositories holding the `ruby-runtime` and Python SDK source code will be archived on GitHub.
 
-The multiple Git repositories holding the `ruby-runtime` source code will be archived on GitHub.
+Support for Ruby Runtime in the Jenkins core will be removed in future Jenkins versions.
```

Comment:
This line is committing core development as part of this JEP. Is this intentional? Who's going to do that?

This is very different from saying we may remove it, or something similarly less committing.

Reply:
```suggestion
Support for Ruby Runtime in the Jenkins core may be removed in future Jenkins versions.
```

I think we agreed to do it in the dev list, but happy to change it to a less-commitment statement. Happy to take the action item though